src/UnikraftExtension.ts

```diff
@@ -82,79 +94,69 @@ export class UnikraftExtension {
             }
         });
 
+    }
+
+    private initExtension() {
+        this.registerCommands();
         this.kraftStatusBarItem.text = 'Unikraft';
         vscode.window.showInformationMessage(
             'Congratulations, your extension "Unikraft" is now active!'
         );
     }
 
     private async setupExtension() {
-        var githubToken = utils.getGithubToken();
-    
-        if (githubToken === '') {
-            this.kraftStatusBarItem.text = 'Waiting for the Github Token...';
-            githubToken = await utils.updateGithubToken(this.kraftChannel);
-        }
-    
-        if (utils.getUkWorkdir() === '') {
-            utils.updateUkWorkdir(path.join(os.homedir(), '.unikraft'));
+        if (utils.getUkPackWorkdir() === '') {
+            utils.updateUkPackWorkdir(path.join(os.homedir(), '.local/share/kraftkit'));
         }
-    
-        if (!fs.existsSync(path.join(os.homedir(), '.kraftrc'))) {
-            this.installKraft(githubToken);
+
+        if (!fs.existsSync('/usr/local/bin/kraft')) {
```

Comment:
I think we should not use absolute paths, maybe people have it installed somewhere else

Reply:
I agree, Found a solution :)

Reply:
Done 👍🏽